examples/03_connectivity/plot_data_driven_parcellations.py

```diff
@@ -200,28 +192,28 @@
 ##################################################################
 # Brain parcellations with ReNA Clustering
 # ----------------------------------------
+#
 # The spatial constraints are implemented inside the ReNA object.
+# One interesting algorithmic property of ReNA is that it is very
+# fast for a large number of parcels (notably faster than Ward).
 #
-# We use caching. As a result, the clustering doesn't have
-# to be recomputed later.
-from nilearn.regions import ReNA
-start = time.time()
-rena = ReNA(scaling=True, n_clusters=1000, memory='nilearn_cache')
-
-rena.fit_transform(dataset.func[0])
-print("ReNA 1000 clusters: %.2fs" % (time.time() - start))
+# References
+# ----------
+#
+# More about ReNA clustering algorithm in the original paper
+#
+#     * A. Hoyos-Idrobo, G. Varoquaux, J. Kahn and B. Thirion, "Recursive
+#       Nearest Agglomeration (ReNA): Fast Clustering for Approximation of
+#       Structured Signals," in IEEE Transactions on Pattern Analysis and
+#       Machine Intelligence, vol. 41, no. 3, pp. 669-681, 1 March 2019.
+#       https://hal.archives-ouvertes.fr/hal-01366651/
 
+from nilearn.regions import ReNA
 start = time.time()
-rena = ReNA(scaling=True, n_clusters=2000, memory='nilearn_cache')
+rena = ReNA(n_clusters=5000, scaling=True)
```

Comment:
I would prefer if we used the "Parcellation" object, with method="rena".

Reply:
Yes that is what I meant by 

> Integrate ReNA in Parcelation, update the examples accordingly